drivers/hmis/app/models/hmis/ce/match/candidate_pool_builder.rb

```diff
@@ -0,0 +1,78 @@
+# update candidate pools to reflect the current opportunities, requirements and priority configuration
+
+module Hmis::Ce::Match
+  class CandidatePoolBuilder
+    # assumes a relatively small number of active opportunities
```

Comment:
We should probably define "relatively small" (is it < 10, < 100, or < 1,000?)
Also, maybe a quick note about what might need to be adjusted to handle more.  I expect we'll quickly end up with > 100 opportunities.